charts/argo-cd/README.md.gotmpl

```diff
@@ -104,6 +104,12 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.35.0
+This version supports Kubernetes version `>=1.23.0-0`. Though the supported version of Kubernetes is v1.24 or later, we align with Amazon EKS because there are a lot of EKS users.
```

Comment:
How about something like this? 
```suggestion
This version supports Kubernetes version `>=1.23.0-0`. The current supported version of Kubernetes is v1.24 or later and we align with Amazon EKS calendar, because many of AWS users and conservative approach.
```